ZAPD/ZRoom/Commands/SetActorCutsceneList.cpp

```diff
@@ -0,0 +1,97 @@
+#include "SetActorCutsceneList.h"
+#include "../../BitConverter.h"
+#include "../../Globals.h"
+#include "../../StringHelper.h"
+#include "../../ZFile.h"
+#include "../ZRoom.h"
+
+using namespace std;
+
+SetActorCutsceneList::SetActorCutsceneList(ZRoom* nZRoom, std::vector<uint8_t> rawData, int rawDataIndex)
+	: ZRoomCommand(nZRoom, rawData, rawDataIndex)
+{
+	int numCutscenes = rawData[rawDataIndex + 1];
+	segmentOffset = BitConverter::ToInt32BE(rawData, rawDataIndex + 4) & 0x00FFFFFF;
+
+	cutscenes = vector<ActorCutsceneEntry*>();
+
+	int32_t currentPtr = segmentOffset;
+
+	for (int i = 0; i < numCutscenes; i++)
+	{
+		ActorCutsceneEntry* entry = new ActorCutsceneEntry(rawData, currentPtr);
+		cutscenes.push_back(entry);
+
+		currentPtr += 16;
+	}
+	
+	string declaration = "";
+
+	for (ActorCutsceneEntry* entry : cutscenes)
+	{
+		declaration +=
+			StringHelper::Sprintf("\t{ %i, %i, %i, %i, %i, %i, %i, %i, %i, %i },\n", entry->priority,
```

Comment:
We try to use 4 spaces instead of a tab for the output.
Please change, unless the MM codebase use tabs instead of spaces too.
```suggestion
			StringHelper::Sprintf("    { %i, %i, %i, %i, %i, %i, %i, %i, %i, %i },\n", entry->priority,
```

Reply:
hopefully in the future we will be able to use OutputFormatter to do this automatically. that way we can use \t in the code which looks a bit cleaner and then change it to spaces later